### PR TITLE
V1.0.3

### PR DESCRIPTION
**Pull request by SC426**
[Upstream](https://github.com/RomeoDespres/pkce)

## Actions
- Forked from Conda Forge
- Remove `noarch` build
- A bunch of linter fixes (see commits below)

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,16 +10,18 @@ source:
   sha256: 9775fd76d8a743d39b87df38af1cd04a58c9b5a5242d5a6350ef343d06814ab6
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [py<3.0]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<36]
```
Skipping py<36 will be better as versions 3.0-3.5 are not used anymore

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,16 +10,18 @@ source:
...
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
```
Comment (rafaelmartins) [inline recipe/meta.yaml:15]:
```suggestion
  number: 1
```
as version is not bumped, build number must

Comment (skupr-anaconda) [reply recipe/meta.yaml:15]: It's a new package so the build number `0` is correct

Comment (rafaelmartins) [reply recipe/meta.yaml:15]: ok

**Review by skupr-anaconda — APPROVED** [review]

**Review by rafaelmartins — APPROVED** [review]

**Pull request merged**